Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -12,6 +12,38 @@ PARAMETERS:
 "
 }
 
+while [[ $# > 1 ]]
+do
+  key="$2"
+  case $key in
+    -h|--help)
+      usage
+      exit
+      ;;
+    -e|--eom)
+      EOM="$3"
+      shift
+      ;;
+    -E|--eop)
+      EOP="$3"
+      shift
+      ;;
+    -B|--eob)
+      EOB="$3"
+      shift
+      ;;
+    -*)
+      echo "(ERROR) Unknown option: $key" >&2
+      usage >&2
```

Comment:
As you have added command line parameters, it would be nice to reflect in the usage message. Or else user might stumble like this:
```
$ ./load_data.sh filename -b 1           
(ERROR) Unknown option: -b
USAGE:
load_data.sh [FILE...]

PARAMETERS:
  FILE -- file in NDJSON format for loading to Elasticsearch via bulk interface
```
See, the message that appears after error message does not explain how to use the script properly. The only way to get this information is to read the source code, which is not good.